Making Sense/Interface/IMethods.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Making_Sense.Interface.IMethods
+{
+    public interface IMethods
```

Comment:
Qué te parece si usamos un nombre mas representativo y no tan abierto? Se me ocurre algo como lo siguiente:
```suggestion
    public interface ICarCrud
```

Reply:
Dale! Gracias Darian